packages/sanity/src/structure/panes/document/documentPanel/banners/__telemetry__/DraftLiveEditBanner.telemetry.ts

```diff
@@ -0,0 +1,21 @@
+import {defineEvent} from '@sanity/telemetry'
+
+/**
+ * When a draft in a live edit document is published
+ * @internal
+ */
+export const PublishedLiveEditDraft = defineEvent({
+  name: 'Publish a draft (live edit)',
```

Comment:
A question for growth mainly I guess: do we want to be more clear about context here? E.g. "Resolve liveEdit by publishing draft"

Reply:
I'm ok with making that change :)